### PR TITLE
added alpas

### PR DESCRIPTION
**Pull request by shresritik**
## What kind of change does this PR introduce? 

- [ ] Bug Fix
- [ ] Documentation change
- [x] Added new company
- [ ] Other(s)

## Why was this change needed?
added alpas technology data

### CODE REVIEW

**Review by TheLearneer — CHANGES_REQUESTED** [review]
Please make the fixes 🙏

File: data/companies.json [hunk 1]
```diff
@@ -543,6 +543,22 @@
         "twitter": "https://twitter.com/yarsalabs/",
         "youtube": "https://www.youtube.com/channel/UCqJAqxSBEmpdCqwgQG3Qoeg"
       }
+    },
+    {
+      "name": "https://www.alpas.com.np/",
+      "establishedAt": 1514764800,
+      "location": "Sai Marg, Pulchowk-3, Lalitpur, Nepal",
+      "website": "https://www.yarsa.io/",
```
[the remark this answers is not in the document]
Comment (shresritik) [reply data/companies.json]: Ok, I have corrected the company name. Please check it now

**Review by TheLearneer — APPROVED** [review]
LGMT

**Comment by shresritik** [comment]
Please merge and close the PR

**Pull request merged**